pkg/operation/botanist/component/metricsserver/metrics_server_test.go

```diff
@@ -356,8 +356,8 @@ spec:
           periodSeconds: 10
         resources:
           limits:
-            cpu: 80m
-            memory: 400Mi
+            cpu: 500m
```

Comment:
As the VPA now does not matter, please rename variable names
- `deploymentYAMLWithVPAWithHostEnv` -> `deploymentYAMLWithHostEnv`
- `deploymentYAMLWithoutVPAWithoutHostEnv` -> `deploymentYAMLWithoutHostEnv`